Docs/source/install/hpc/lxplus.rst

```diff
@@ -0,0 +1,139 @@
+.. _building-lxplus:
+
+LXPLUS (CERN)
+=============
+
+The LXPLUS cluster is located at CERN.
+
+* `Lxplus documentation <https://lxplusdoc.web.cern.ch>`__
+* Batch system: `HTCondor <https://batchdocs.web.cern.ch/index.html>`__
+* Filesystem locations:
+    * User folder: ``/afs/cern.ch/user/<a>/<account>`` (10GByte)
+    * Work folder: ``/afs/cern.ch/work/<a>/<account>`` (100GByte)
+    * Eos storage: ``/eos/home-<a>/<account>`` (1T)
+
+Through LXPLUS we have access to CPU and GPU nodes (the latter equipped with NVIDIA V100 and T4 GPUs).
+
+Installation
+------------
+Only very little software is pre-installed on LXPLUS so we show how to install from scratch all the dependencies using `Spack <https://spack.io>`__.
+
+For size reasons it is not advisable to install WarpX in the ``$HOME`` directory, while it should be installed in the "work directory". For this purpose we set an environment variable with the path to the "work directory"
+
+.. code-block:: bash
+
+    export WORK=/afs/cern.ch/work/${USER:0:1}/$USER/
+
+We clone WarpX in ``$WORK``:
+
+.. code-block:: bash
+
+    cd $WORK
+    git clone https://github.com/ECP-WarpX/WarpX.git warpx
+
+Installation profile file
+^^^^^^^^^^^^^^^^^^^^^^^^^
+The easiest way to install the dependencies is to use the pre-prepared ``warpx.profile`` as follows:
+
+.. code-block:: bash
+
+    cd $WORK
+    cp warpx/WarpX/Tools/machines/lxplus-cern/lxplus_warpx.profile.example .
+    source lxplus_warpx.profile.example
```

Comment:
Let's remove the `.example` :) We add this so people know that they might need to edit this file (not the case here).
```suggestion
    cp $WORK/warpx/WarpX/Tools/machines/lxplus-cern/lxplus_warpx.profile.example $WORK/lxplus_warpx.profile
    source $WORK/lxplus_warpx.profile
```

Reply:
Alright, I was actually wondering why it was called example :)